website/docs/faqs/cloud-upgrade-instructions.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Upgrading your dbt Cloud Account"
```

Comment:
Could we redirect this article to the one above and delete this article since they are very similar?

Reply:
i dont know why this showed up here, this is the old, original file that i made changes to. I'm going to delete it to see